src/main/java/graphql/nadel/result/ResultComplexityAggregator.java

```diff
@@ -0,0 +1,45 @@
+package graphql.nadel.result;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+public class ResultComplexityAggregator {
+    private int totalNodeCount = 0;
+    private LinkedHashMap serviceNodeCounts = new LinkedHashMap<String, Integer>();
+
+    public int getTotalNodeCount() {
+        return totalNodeCount;
+    }
+
+    public LinkedHashMap getServiceNodeCounts() {
+        return serviceNodeCounts;
+    }
+
+    public int getNodeCountsForService(String serviceName) {
+        return (int) serviceNodeCounts.get(serviceName);
```

Comment:
AutoBoxing in Java will handle this - no need for the cast

Again IDEA squiggley code inspections should have caught this